ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -636,4 +651,14 @@ FAN_SS_VSC_EPC_EHHRC_EHHRC3X:
   uses:
   - heat_recovery_water_isolation_valve_command_1
   - heat_recovery_water_isolation_valve_command_2
-  - heat_recovery_water_isolation_valve_command_3
\ No newline at end of file
+  - heat_recovery_water_isolation_valve_command_3
+  
+FAN_EFSS_FDPM_EFVSC_SRC: 
+  description: "Exhaust fan with power, voltage, current, differential pressure and run durating monitoring ."
+  is_canonical: true
+  implements:
+  - FAN
+  - EFSS
```

Comment:
SS, not EFSS

Reply:
Changed from EFSS to SS in the canonical. Added run_time_accumulator as optional in the  SS abstract.